__tests__/unit/cloudstore.ios.test.js

```diff
@@ -1,7 +1,7 @@
 import mockAsyncStorage from '@react-native-community/async-storage';
 import * as CloudStore from '../../src/cloudstore';
 
-describe('CloudStore unit test', () => {
+describe('CloudStore ios unit test', () => {
```

Comment:
Remove this diff as `CloudStore` unit test should stay platform agnostic.

Reply:
Updated 🚧